src/Resources/views/Block/block_search_result.html.twig

```diff
@@ -31,8 +31,8 @@ file that was distributed with this source code.
                 </h3>
 
                 <div class="box-tools pull-right">
-                    {% if pager and pager.getNbResults() > 0 %}
-                        <span class="badge">{{ pager.getNbResults() }}</span>
+                    {% if pager and (pager.countResults is defined ? pager.countResults() : pager.getNbResults()) > 0 %}
```

Comment:
is there a more proper way to check if a method exists in twig? :thinking:

Reply:
This I think: https://twig.symfony.com/doc/2.x/functions/attribute.html

You should also add everytime a `NEXT_MAJOR` comment to explain what to do in master branch.